index.html

```diff
@@ -3,13 +3,13 @@
 ---
 <h1>{{ page.title }}</h1>
 
-<p>We are excited to announce that the FedRAMP Tailored baseline is available for public comment. The public comment period ends on <em>March 17, 2017</em> -- <strong>we welcome your feedback</strong>.</p>
+<p>We are excited to announce that the <a href="{{ site.baseurl }}/policy/">FedRAMP Tailored baseline</a> is available for public comment. The public comment period ends on <em>March 17, 2017</em> -- <strong>we welcome your feedback</strong> by <a href="mailto:info@fedramp.gov?subject=Comments+on+FedRAMP+Tailored" onclick="ga('send', 'event', 'Clicked top mailto: link', 'Clicked top mailto: link');">email</a> or <a href="https://github.com/gsa/fedramp-tailored/issues/new" onclick="ga('send', 'event', 'Clicked top GitHub link', 'Clicked top GitHub link');">on GitHub</a>. All comments received will be posted publicly.</p>
```

Comment:
I linked to `/policy/` for the first mention of the baseline, and added click events for GA to the two new links.